packages/testing/src/index.js

```diff
@@ -5,7 +5,4 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-export { default as mountWithTheme } from './utils/mountWithTheme.js';
-export { default as renderWithTheme } from './utils/renderWithTheme.js';
-export { default as shallowWithTheme } from './utils/shallowWithTheme.js';
 export { default as getExports } from './utils/getExports.js';
```

Comment:
Does it make sense to move this one to `react-utilities` and dump the testing package?

Reply:
This is really custom to just us. I would vote to move this to our `garden-test-utils` module mapping along with `react-testing-library`.

Reply:
Yes, even better.

Reply:
I have moved this utility and removed the `react-testing` package 🎉